File: array.cpp
// C++: Calculate the sum of an array with syntax errors
#include <iostream>
using namespace std;

int calculateSum(int arr[], int size) {
    int total = o;  // Error: 'o' should be '0'
    for (int i = o; i < size; i++) {   // Error: 'o' should be '0'
        total += arr[i];
    }
    return total;
}

int main () {
    int numbers [] = {1, 2, 3, 4, 5};
    int size = sizeof(numbers) / sizeof(numbers[o]);
    int result = calculateSum(numbers, size);
    cout << "Sum in C++" " << result << endl; // Error: missing concatenation operator
    return o;
}

File: array.py
# Python: Calculate the sum of an array with syntax errors
def calculate_sum(arr) :
    total = o  # Error: 'o' is a typo, should be '0'
    for num in arr:
        total += num
    return total

numbers = [1, 2, 3, 4, 5]
result = calculate_sum (numbers)
print("Sumin Python :", result)
